src/main.rs

```diff
@@ -102,8 +102,9 @@ const RETRY_BASE_MS: u64 = 8;
 const RETRY_FACTOR: u64 = 1;
 const RETRY_COUNT: usize = 5;
 
-#[tokio::main]
-async fn main() -> anyhow::Result<()> {
+// "the Sentry client must be initialized before starting an async runtime or spawning threads"
+// https://docs.sentry.io/platforms/rust/#async-main-function
+fn main() {
```

Comment:
Optional: this can still return a `anyhow::Result<()>` to leverage the try sigil (`?`)